internal/agent/component/io/xlsx_writer.go

```
@@ -0,0 +1,284 @@
+	if strings.TrimSpace(name) == "" {
+		name = fmt.Sprintf("Table_%d", index+1)
+	}
+	return name
+}
+
+// WriteXLSX converts Markdown content into an Excel workbook: each
+// Markdown table lands on its own sheet; when the content has no
+// table at all, a single "Data" sheet holds the whole text.
+func WriteXLSX(content string, _ XLSXOptions) ([]byte, error) {
+	f := excelize.NewFile()
+	defer f.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `f.Close` is not checked

(errcheck)